### PR TITLE
selenium-server-standalone 2.46.0

### PR DESCRIPTION
**Pull request by rubytester**

### CODE REVIEW

File: Library/Formula/selenium-server-standalone.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class SeleniumServerStandalone < Formula
   desc "Browser automation for testing purposes"
   homepage "http://seleniumhq.org/"
-  url "http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar"
-  sha1 "9bc872d1f364a3104257b1f8e055a342228259c3"
+  url "http://selenium-release.storage.googleapis.com/2.46/selenium-server-standalone-2.46.0.jar"
```
Comment (vszakats) [inline Library/Formula/selenium-server-standalone.rb]: Please use https here.

File: Library/Formula/selenium-server-standalone.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  sha1 "679ab7340901ea72dd32b26f0eacc4619330ad79"
```
Comment (vszakats) [inline Library/Formula/selenium-server-standalone.rb]: And sha256 hash here. Thank you!

**Comment by rubytester** [comment]
@vszakats  Thanks for comments. I amended with https and sha256

**Comment by vszakats** [comment]
@rubytester: Thank you!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/27f906427b1e9e4b7ac9d1822f1d02e375ee1011. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**